target/linux/ath79/files/arch/mips/include/asm/mach-ath79/rb4xx_cpld.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This is a direct copy of https://github.com/openwrt/openwrt/blob/master/target/linux/ar71xx/files/arch/mips/include/asm/mach-ath79/rb4xx_cpld.h